### DISCUSSION TITLE
emergency access flow?

### DISCUSSION THREAD

**Discussion by tessus** [dani-garcia/vaultwarden#2906] [Q&A]
Someone has added me as their emergency access, but I have no idea how to request access. It is currently not necessary, but I am just trying to figure this out **before** it becomes necessary.

![image](https://user-images.githubusercontent.com/223439/201010291-c14c383a-b262-4543-8b6a-0d947a6dd6a5.png)

As you can see there's only the option to `Remove` myself as an emergency access contact, but not to request access. Or did I hit a bug? (I'm using vw 1.26.0 and webvault 2022.10.2)

**Comment by tessus** [comment]
Ok. I have tested this now on a test installation.

The flow is a bit off. Even after a user is invited and that user has accepted to be an emergency contact, the original user still has to confirm that user. IMO the last step is useless.

I have thus changed my original question of this discussion.

Can someone please explain to me why it is necessary to confirm a user, even after I have designated (invited) this user as an emergency contact?

**Reply by BlackDex** [reply]
Because you could have made a typo. During the confirm it asks for some verification code, which the requester can ignore if they want.

I would want to have something like this double checked before i give someone access to all my passwords. This is the same flow as for inviting someone to an organization.

But, if you really want to have a very good answer on why this flow is like this, i suggest to ask this question over at the Bitwarden community forums.

**Reply by tessus** [reply]
Haha. The typo is supposed to be what? An entire email address?

> This is the same flow as for inviting someone to an organization.

Yep, I never understood that one either.

> i suggest to ask this question over at the Bitwarden community forums.

Will do. Thanks.
